pkg/operator/starter.go

```diff
@@ -64,6 +71,12 @@ func RunOperator(ctx context.Context, controllerConfig *controllercmd.Controller
 		return err
 	}
 
+	controlPlaneDynamicClient, err := dynamic.NewForConfig(rest.AddUserAgent(controlPlaneKubeConfig, targetName))
+	if err != nil {
+		return err
+	}
+	controlPlaneDynamicInformers := dynamicinformer.NewDynamicSharedInformerFactory(controlPlaneDynamicClient, 12*time.Hour)
```

Comment:
The informer should be used in controllerDeploymentController, https://github.com/openshift/cluster-csi-snapshot-controller-operator/blob/a6f68bc70191b6a67aa49f955249f9ccf1354fd8/pkg/operator/starter.go#L215-L218